src/dlfcn.c

```diff
@@ -48,6 +49,7 @@
 #ifdef SHARED
 #define DLFCN_WIN32_EXPORTS
 #endif
+#define __USE_GNU

```

Comment:
Why it is needed?

Reply:
This is left over from a previous remove